### ISSUE TITLE
long_description breaks pip install in Python 3.6

### ISSUE THREAD

**Issue by mauricemeyer** [inducer/gmsh_interop#2]
On Ubuntu 18.04 with Python 3.6, `pip install` won’t work because the Readme contains non-ASCII characters.

This is in [Line 16](https://github.com/inducer/gmsh_interop/blob/8b0f33100ee1a1ac2e300094fe32446580bddd02/setup.py#L16):

```
long_description=open("README.rst", "r").read(),
```

`pip install` fails with:

```
Collecting gmsh_interop==2017.1
  Using cached https://files.pythonhosted.org/packages/ed/f6/e442f6346b04c565262cf7b12677a4b1a41cc1ed1f272aef912a32ce25cc/gmsh_interop-2017.1.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-1l_ceng9/gmsh-interop/setup.py", line 16, in <module>
        long_description=open("README.rst", "r").read(),
      File "/root/c3nav/c3nav/env/lib/python3.6/encodings/ascii.py", line 26, in decode
        return codecs.ascii_decode(input, self.errors)[0]
    UnicodeDecodeError: 'ascii' codec can't decode byte 0xc3 in position 637: ordinal not in range(128)

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-1l_ceng9/gmsh-interop/
```

**Comment by inducer** [comment]
Thanks for the report. I wasn't able to reproduce it on my system (Debian testing/10), but removing the offending character isn't hard. This should help. Let me know if not.

**Comment by inducer** [comment]
Released 2019.1 with this fix.
